src/components/home/employees/list/EmployeeList.js

```diff
@@ -56,7 +57,7 @@ class EmployeeList extends Component {
   }
 
   componentDidMount() {
-    this.fetchEmployees();
+    this.fetchEmployees({});
```

Comment:
instead of passing an empty object, use a default value in `fetchEmployees` function